pyfakefs/fake_pathlib.py

```diff
@@ -131,9 +132,17 @@ def lchmod(self, pathobj, mode):
 
     if sys.version_info >= (3, 9):
         readlink = _wrap_strfunc(FakeFilesystem.readlink)
+    else:
+        # why? and shouldnt this not be *os* -- but something patched (FakeOsModule probably)?
+        readlink = staticmethod(os.readlink)
+        pass
 
     utime = _wrap_strfunc(FakeFilesystem.utime)
 
+    # same comment as above for these two
+    realpath = staticmethod(os.path.realpath)
+    getcwd = staticmethod(os.getcwd)
+
```

Comment:
this and the above section I am 99% sure I am doing something wrong/bad here -- but the gist of this is it looks like we needed to add some methods to the FakeAccessor object -- I am *assuming*(?) its not a great idea to just attach `os` methods here because we would want to have the accessor using the patched methods? But this does seem to get more of the tests passing so figured id commit it to try to be as helpful as I can... (even if it isn't much :))

Reply:
I changed it accordingly. Regarding `getcwd` - if using `os` inside the function, as I do it now, it will be patched, so it will behave correctly.

Reply:
I removed `realpath` as it doesn't seem to be really needed, but maybe I'm wrong here.